app/views/commodities/_commodity.html.erb

```diff
@@ -23,7 +23,7 @@
           <% end %>
         </div>
         <div class="duty" aria-describedby="commodity-duty-title">
-          <% tcd = commodity.overview_measures.third_country_duty %>
+          <% tcd = commodity.overview_measures.third_country_duties %>
```

Comment:
I think this shifts from using a MeasureCollection to using a Array - not sure that makes any difference though?

Reply:
I don't think it actually matters for the overview measures. I believe they're also not behind the MeasurePresenter :shrug:

Reply:
Probably better for consistency, though